### ISSUE TITLE
alt text

### ISSUE THREAD

**Issue by marmercurio** [BirkbeckCTP/janeway#2825]
I suggest that there be a way for Janeway to capture alt text associated with a particular image at the submission stage—or at least prior to the alt text's encoding in XML of the article. Essentially, I would like a place—either associated with an uploaded file or elsewhere—for authors to paste in or upload their alt text for those images. We're hoping it would make it easier for editors to gather data and its presence would impel authors to provide it. Thank you.

**Comment by StephDriver** [comment]
for discussion: Alt text is specific to the image within its context
1. if an uploaded images are only ever used in once?   then image: alt is a 1:1 relationship and it can be captured once.
2. if there are uploaded images that may be used more than once, then the image: alt is a 1:many relationship, and it cannot be input just once.

**Comment by ajrbyers** [comment]
Discussed in the a11y meeting:

1. For each kind of image in Janeway we need to identify them and their agent
2. Combine warnings with documentation to assist users with alt text (e.g. for XML and HTML warn the user when alt text is missing, and how they can fix it)
3. For Janeway images that are not galleys discuss how this model will need to work

**Comment by StephDriver** [comment]
- [ ] alt text for journal cover images should also be considered.

from BirkbeckCTP/hourglass#324

**Comment by StephDriver** [comment]
Linking in the need for an audit of what alt-text is missing as per #3838

**Comment by severett89** [comment]
Reminder that this ties into OLHJ - potential mention in author guidelines but also OLH wide image reproduction/permissions/use guide that I am putting together.

**Comment by StephDriver** [comment]
Note, maths content also needs alternative text as an aria-label and it may be worth considering alongside image alt-text, see #3923

**Comment by ajrbyers** [comment]
From backlog: As part of this work we should consider also checking for missing aria-labels on math content (https://github.com/openlibhums/janeway/issues/3923)

**Comment by mauromsl** [comment]
closed by https://github.com/openlibhums/hourglass/issues/471

**Comment by joemull** [comment]
Closed by mistake?

**Comment by mauromsl** [comment]
BRF Notes:

1. As an editor, when configuring the submission system I can toggle a field for authors to provide an alt text for an image. It should be worded in a way that non technical users can understand.
2. As an author I can provide an alt text to all my image and supllementary file uploads
3. during copyediting, as an editor, I can see and edit the alt text provided for each file
4. during typesetting, both as an editor and a typesetter, I can see the alt text provided
5. as a copyeditor, I can see (and make changes?) to alt text.
